lib/elixometer.ex

```diff
@@ -345,11 +345,12 @@ defmodule Elixometer do
       cfg = Application.get_all_env(:elixometer)
       reporter = cfg[:reporter]
       interval = cfg[:update_frequency]
+      extra = cfg[:extra] || true
 
       if reporter do
         :exometer.info(metric_name)
         |> Keyword.get(:datapoints)
-        |> Enum.map(&(:exometer_report.subscribe(reporter, metric_name, &1, interval)))
+        |> Enum.map(&(:exometer_report.subscribe(reporter, metric_name, &1, interval, extra)))
```

Comment:
Yeah... `true` is the default value for `extra` coming from exometer, but if we wanna specifically call the `subscribe/4 `version of the function I can change it to that. Let me know.